static/menu.js

```diff
@@ -68,15 +68,15 @@ async function loadAjaxMenu(menu, id, msg) {
         } else if (currentMenu.querySelector(".pure-menu-link:focus")) {
             currentMenu.firstElementChild.focus();
         }
-        currentMenu.className = currentMenu.className.replace("pure-menu-active", "");
+        currentMenu.classList.remove("pure-menu-active");
```

Comment:
I also did some refactoring here changing all class manipulation/testing to go through `classList` after I noticed some of these.